source/common/http/http1/conn_pool.h

```diff
@@ -43,6 +43,7 @@ class ConnPoolImpl : public ConnectionPool::Instance, public ConnPoolImplBase {
   bool hasActiveConnections() const override;
   ConnectionPool::Cancellable* newStream(StreamDecoder& response_decoder,
                                          ConnectionPool::Callbacks& callbacks) override;
+  Upstream::HostDescriptionConstSharedPtr host() const override { return host_; };
```

Comment:
Can you add a test for this and the HTTP/2 one in that UTs for the conn pools?

Reply:
Good catch, done.